### PR TITLE
Applying a site selector to the content selector

### PR DESCRIPTION
**Pull request by hbaptiste**
Applying a site selector to the content selector

### CODE REVIEW

File: src/tb/component/contentselector/main.js [hunk 1]
```diff
@@ -444,7 +462,7 @@ define(
                     } else {
                         selector.loadRootNode();
                     }
-                    selector.isLoaded = true;
+                    //selector.isLoaded = true;
```
Comment (mickaelandrieu) [inline src/tb/component/contentselector/main.js]: if you don't need it, you can remove it :)

**Comment by mickaelandrieu** [comment]
:+1:  for me, good job @hbaptiste

**Comment by hbaptiste** [comment]
almost, I have one label to translate. My bad. Hold on!

**Comment by mickaelandrieu** [comment]
^_^

**Comment by ndufreche** [comment]
@hbaptiste can you remove commented code please, then I will merge this PR.

File: src/tb/component/contentselector/main.js (hunk 1 above)
Comment (mickaelandrieu) [inline src/tb/component/contentselector/main.js]: @hbaptiste its not correct yet, remove this line please

**Comment by mickaelandrieu** [comment]
thanks @hbaptiste

**Pull request merged**